stable/prometheus-blackbox-exporter/templates/deployment.yaml

```diff
@@ -85,6 +85,18 @@ spec:
           volumeMounts:
             - mountPath: /config
               name: config
+          {{- range .Values.extraConfigmapMounts }}
+            - name: {{ .name }}
+              mountPath: {{ .mountPath }}
+              subPath: {{ .subPath | default "" }}
```

Comment:
Sorry, I missed it: if it is optional, put a if condition to set the line or skip it (`mountPath: ` will not validate against kubernetes json schemas)

Reply:
Currently the subpath is already optional, since if you give it an empty value (or don't mention it) in the values.yaml, the kubernetes json schema will give it a null value.